ImageBuilderRAM/ImagePipeline_AppAccount_Sample_AmznLinux.yml

```diff
@@ -0,0 +1,215 @@
+AWSTemplateFormatVersion: 2010-09-09
+
+Parameters:
+  IamEC2IBRoleName:
+    Description: IAM Role for EC2ImageBuilder Instance Profile to be attached to EC2 used as BuildServer
+    Type: String
+    Default: EC2ImageBuilderRole
+
+  ImageBuilderSNSTopicName:
+    Description: ImageBuilder SNS Topic Name
+    Type: String
+    Default: "ImageBuilderNotificationsTomCat"
+  
+  NotificationEmail:
+    Description: Email address to receieve notifications
+    Type: String
+    Default: "email@example.com"
+ 
+  SubnetId:
+    Description: Subnet Id that will be used to spin up the build server needs access to public internet 
+    Type: AWS::EC2::Subnet::Id
+    
+  SecurityGroupIds:
+    Description: Security Group Ids that will be attached to build server
+    Type: List<AWS::EC2::SecurityGroup::Id>
+    
+  InstanceTypes:
+    Type: CommaDelimitedList
+    AllowedValues:
+      - t3.small
+      - t3.medium
+      - c5.large
+      - m5.large
+      - r5.large
+
+  SharedAccountId:
+    Type: String
+    AllowedPattern: ^\d{12}$
+    ConstraintDescription: 12 Digit Account number of the Shared Account
+    Default: 123456789100
+  
+  KMSKeyArn:
+    Type: String
+    Description: Key used for Encrypting Image in Shared Account
+    Default: arn:aws:kms:us-east-1:xxxxxxxx:key/xxxxxx-2b78-4731-bfbb-xxxxxxxxx
+
+Resources:
+
+  EC2IBRole:
+    Metadata:
+      cfn-lint: { 
+        config: { 
+          ignore_checks: [ EIAMPolicyWildcardResource, EIAMPolicyActionWildcard ],
+          ignore_reasons: [ 
+            EIAMPolicyWildcardResource: "Requires full access to function",
+            EIAMPolicyActionWildcard: "Required in order to properly function"
+          ]
+        }
+      }
+    Type: AWS::IAM::Role
+    Properties:
+      RoleName: !Ref IamEC2IBRoleName
+      Path: /
+      AssumeRolePolicyDocument:
+        Version: 2012-10-17
+        Statement:
+          - Effect: Allow
+            Principal:
+              Service:
+              - ec2.amazonaws.com
```

Comment:
Use url suffix pseudo parameter instead of hard coding ec2.amazonaws.com, to ensure compatibility with other AWS partitions which may use a different suffix.

`- !Sub "ec2.${AWS::URLSuffix}"`